src/dotnet/devcontainer-feature.json

```diff
@@ -34,7 +34,7 @@
     "customizations": {
         "vscode": {
             "extensions": [
-                "ms-dotnettools.csharp"
+                "ms-dotnettools.csharp@prerelease"
```

Comment:
We have always added support for extensions which are production ready (ie. in release phase). I don't believe we have added any extension in a `prerelease` phase. 🤔 

> To use the C# Dev Kit today, users need the pre-release version of the C# extension.

I understand that its needed for `C# Dev Kit `, however, prerelease versions are generally used for pre-production/testing and might contain bugs. Not sure, if we should recommend them to users 🤔 

Wondering if we could simply wait until the changes reached prod? 
@bamurtaugh What do you think?

Reply:
Thanks for the tag -- yeah I agree...in 'official' dev container it probably isn't right to do this yet. We're trying to rapidly exit pre-release as best we can.  Target is end summer.

Reply:
Thanks for the thoughts! I'll go ahead and close this and https://github.com/devcontainers/images/pull/619.